package/helm/crater/values.yaml

```diff
@@ -23,5 +23,24 @@ service:
     type: ClusterIP
     port: 9000
 
+config:
+  APP_URL: http://crater.test
+  DB_DATABASE: crater
+  SANCTUM_STATEFUL_DOMAINS: crater.test
+  SESSION_DOMAIN: crater.test
+  AUTO_INSTALL: "true"
+  ADMIN_NAME: Jane Doe
+  COMPANY_NAME: xyz
+  COMPANY_SLUG: xyz
+  COUNTRY_ID: 1
+
+secrets:
+  DB_HOST: ""
+  DB_PORT: ""
+  DB_USERNAME: ""
+  DB_PASSWORD: ""
+  ADMIN_EMAIL: ""
+  ADMIN_PASSWORD: ""
```

Comment:
only passwords are secrets

Reply:
So, should we move the `DB_HOST`, `DB_PORT`, `DB_USERNAME`, `ADMIN_EMAIL` to config yml?

Reply:
I have moved the `DB_HOST` and `DB_PORT` to the configMap but I'm a bit skeptical about `DB_USERNAME` and `ADMIN_EMAIL` since they are also credentials, just wanted to add a second thought. Please let me know if you still want to move it.